### PR TITLE
.github: add markdownlint in ci

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

Add markdownlint in `ci.yaml`

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v1.1 (TiDB Operator 1.1 versions)
- [x] v1.0 (TiDB Operator 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/docs-tidb-operator/pull/210

### CODE REVIEW

File: zh/configure-a-tidb-cluster.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ category: reference
 aliases: ['/docs-cn/dev/tidb-in-kubernetes/reference/configuration/tidb-cluster/','/docs-cn/v3.1/tidb-in-kubernetes/reference/configuration/tidb-cluster/','/docs-cn/v3.0/tidb-in-kubernetes/reference/configuration/tidb-cluster/']
 ---
 
+<!-- markdownlint-disable MD007 -->
```
Comment (TomShawn) [inline zh/configure-a-tidb-cluster.md:8]:
```suggestion
```

File: zh/configure-a-tidb-cluster.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ category: reference
...
+
 # Kubernetes 上的 TiDB 集群配置
```
Comment (TomShawn) [inline zh/configure-a-tidb-cluster.md:11]: This comment should be placed under the level-1 heading.
```suggestion

<!-- markdownlint-disable MD007 -->

```

File: zh/configure-a-tidb-cluster.md [hunk 3]
```diff
@@ -5,8 +5,11 @@ category: reference
 aliases: ['/docs-cn/dev/tidb-in-kubernetes/reference/configuration/tidb-cluster/','/docs-cn/v3.1/tidb-in-kubernetes/reference/configuration/tidb-cluster/','/docs-cn/v3.0/tidb-in-kubernetes/reference/configuration/tidb-cluster/']
 ---
 
+
```
Comment (ran-huang) [inline zh/configure-a-tidb-cluster.md:8]:
```suggestion
```

**Comment by ran-huang** [comment]
@TomShawn Thanks! Suggestions applied. PTAL again~

**Comment by TomShawn** [comment]
LGTM
@aylei PTAL

**Pull request merged**